src/components/Footer/Footer.js

```diff
@@ -1,9 +1,69 @@
-import "./Footer.css";
+import {Link} from 'react-router-dom';
+import styled from 'styled-components';
 
+import SearchIcon from '../../assets/search-blue.svg';
+/* import WatchlistIcon from '../../assets/watchlist-add.svg';
+ */
 export default function Footer() {
   return (
-    <div className="footer">
-      <h2>This is the footer</h2>
-    </div>
+    <StyledFooter className="footer">
+      <FooterContainer>
+        <Link to="/search">
+          <FooterIconContainer>
+            <FooterIcon src={SearchIcon} alt="" />
+            <FooterIconSearchText>Search</FooterIconSearchText>
+          </FooterIconContainer>
+        </Link>
+        {/*   <Link to="/watchlist">
+          <FooterIconContainer>
+            <FooterIcon src={WatchlistIcon} alt="" />
+            <FooterIconSearchText>Watchlist</FooterIconSearchText>
+          </FooterIconContainer>
+        </Link> */}
```

Comment:
Please remove unused code before merging into main.
```suggestion
```

Reply:
This is also is from the watchlist. Same mistake as above, stupid mistake